### ISSUE TITLE
Custom Buttons

### ISSUE THREAD

**Issue by Nikki-Nik** [stefanwalther/sense-navigation#143]
### Description

Hi - Do we have the ability to add our own button designs? I need a specific colour. I've tried altering the css files but these don't seem to add anything.

Many thanks

### Steps to Reproduce

1. [Step one]
2. [Step two]
3. ...

##### Expected behavior

[What you expected to happen]

##### Actual behavior

[What actually happened]

##### Operating system

```
[ ] Windows
[ ] OSX
[ ] Linux
```
##### Qlik Sense

```
[ ] Desktop
[ ] Enterprise
```

##### Browser

```
[ ] Chrome
[ ] Firefox
[ ] Internet Explorer
[ ] Edge
[ ] Others: XXX
```

### Versions

* Extension version: 
* Browser: 
* Qlik Sense: 
* Operating system: 
* [Other relevant versions]

**Comment by stefanwalther** [comment]
Yes, the new version of _sense-navigation_ supports this, see here:
https://github.com/stefanwalther/sense-navigation/tree/v1.0#custom-styles-custom-css

(BTW: I am planning to release this new version as a v1.0 over the course of summer 2018).

**Comment by stefanwalther** [comment]
As this is included in the #58 version, which I'll release soon, I am closing this request.
